pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -0,0 +1,424 @@
+package testint
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/helpers/random"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_SecurityIntegrations(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	acsURL := fmt.Sprintf("https://%s.snowflakecomputing.com/fed/login", testClientHelper().Context.CurrentAccount(t))
+	issuerURL := fmt.Sprintf("https://%s.snowflakecomputing.com", testClientHelper().Context.CurrentAccount(t))
+
+	cleanupSecurityIntegration := func(t *testing.T, id sdk.AccountObjectIdentifier) {
+		t.Helper()
+		t.Cleanup(func() {
+			err := client.SecurityIntegrations.Drop(ctx, sdk.NewDropSecurityIntegrationRequest(id).WithIfExists(sdk.Pointer(true)))
+			assert.NoError(t, err)
+		})
+	}
+	cert := random.GenerateX509(t)
+	createSAML2Integration := func(t *testing.T, siID sdk.AccountObjectIdentifier, issuer string, with func(*sdk.CreateSaml2SecurityIntegrationRequest)) {
+		t.Helper()
+		_, err := client.ExecForTests(ctx, "ALTER ACCOUNT SET ENABLE_IDENTIFIER_FIRST_LOGIN = true")
```

Comment:
this it still not rolled back and should be used through helper

Reply:
Fixed.